packages/interface/src/index.js

```diff
@@ -1,2 +1,13 @@
 export * from './components';
 export { store } from './store';
+
+export const secretKey = process.env.WP_SECRET_KEY;
```

Comment:
It's possible, probable even, I am totally missing something but wouldn't this lead to the problem of plugins being able to import the secret key? 

I ask this:
* knowing this is a really early POC so maybe I should wait
* acknowledging Adam's earlier comment about the overhead. 

It's not that I don't trust plugins not to import the secret key if it's available, but it [kind of is](https://wpdirectory.net/search/01G3WJTMGBK56JVKPXSFBR562S).

Reply:
The way I understand this is that it solves the problem of experiments not being exposed in `wp.*` JavaScript globals. Plus the key even if you get it, it changes with every release, therefore you'll never be able to rely on it for more than a few months. This is a great leap compared to how `__experimental*` is withing a simple `wp.*` call AND sometimes it remains for years unchanged, therefore "reliable".

Reply:
Thanks both. I was confused between internal exports for the package to use, vs external exports to the `wp` global.

I think I have it now, thanks for your patience explaining.